### DISCUSSION TITLE
Can somebody compile all linux, mac, windows, and html5 builds for version 0.3.2/0.4.1? (any inbetween)

### DISCUSSION THREAD

**Discussion by Beelijah** [FunkinCrew/Funkin#3723] [Q&A] [answered]
# okay, this seems stupid
yeah, it kinda is. but the new versions take a fat ton of space, and i'd like to make a light mod.

# do it yourself
ive been trying for the past 3 days nonstop, but i keep getting error after error.
yes, ive deleted my .haxelib folder and tried again, and even tried using 0.5.2's source, with 0.3.2's assets. but then i realized that was a dumb idea, and it didn't work even when i tried.

# then fix the errors
i have tried, but they never end. its gotten to the point where ive tried editing the original source, and even deleting the files causing the issues.

# where do i upload them?
just use mediafire, or just upload temporarily to google drive, then after about a week you can delete them. if you have a different file sharing service, feel free to use that!

**Comment by moxie-coder** [comment]
why are you trying so hard to use 0.3.2? can you atleast show the errors so people can help you fix them

**Reply by Beelijah** [reply]
0.3.2 doesn't use as much space, and is easier to deal with (in terms of complexity).

**Comment by Beelijah** [comment, marked as answer]
nevermind, found some 0.4.0 builds on itch.io.

edit: he he he, also found some 0.3.3 builds i can use
